.github/workflows/CI_readme_sync.yml

```
@@ -16,8 +16,26 @@ env:
   TAG: ${{ inputs.tag || github.ref_name }}
 
 jobs:
+  get-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      versions: ${{ steps.version_finder.outputs.versions }}
+    steps:
+      - name: Get Haystack Docs versions
+        id: version_finder
+        run: |
+          curl https://dash.readme.com/api/v1/version --header 'authorization: Basic ${{ secrets.README_API_KEY }}' > out
+          VERSIONS=$(jq '[ .[] | select(.version | startswith("2."))| .version ]' out)
+          echo "versions=$VERSIONS" >> "$GITHUB_OUTPUT"
+
   sync:
     runs-on: ubuntu-latest
+    needs: get-versions
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        hs-docs-version: ${{ needs.get-versions.outputs.versions }}
```

I think this is necessary. 🤔 

```suggestion
        hs-docs-version: ${{ fromJSON(needs.get-versions.outputs.versions) }}
```

---

Ah yeah, that's JSON